package.json

```diff
@@ -15,5 +18,18 @@
   "bugs": {
     "url": "https://github.com/apostrophecms/eslint-config-apostrophe/issues"
   },
-  "homepage": "https://github.com/apostrophecms/eslint-config-apostrophe#readme"
+  "homepage": "https://github.com/apostrophecms/eslint-config-apostrophe#readme",
+  "peerDependencies": {
+    "eslint": "^7.1.0"
+  },
+  "dependencies": {
```

Comment:
How does this jive with the current readme?

> Due to limitations of eslint the dependencies of eslint plugins like this one
must be explicitly loaded in your `package.json`.
> Here is the necessary section of `package.json`:
```javascript
"devDependencies": {
    "mocha": "^5.1.1",
    "eslint": "^4.0.0",
    "eslint-config-standard": "^11.0.0",
    "eslint-plugin-import": "^2.13.0",
    "eslint-plugin-node": "^6.0.1",
    "eslint-plugin-promise": "^3.8.0",
    "eslint-plugin-standard": "^3.1.0"
  }
```

Reply:
The readme probably needs to be updated. ESLint recommends doing it that way: https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config

I'm not 100% sure but also don't think they need to include `mocha` in their local package.json since that's just for testing. Semi wondering if this could also be automated...